public/app/features/provisioning/utils/git.ts

```diff
@@ -4,7 +4,7 @@ export function createPRLink(spec?: RepositorySpec, dashboardName?: string, ref?
   if (!spec || spec.type !== 'github' || !ref) {
     return '';
   }
-  return `https://github.com/${spec.github?.owner}/${spec.github?.repository}/compare/${spec.github?.branch}...${ref}?quick_pull=1&labels=grafana&title=Update dashboard ${dashboardName}&body=${encodeURI(comment || '')}`;
+  return `${spec.github?.url}/compare/${spec.github?.branch}...${ref}?quick_pull=1&labels=grafana&title=Update dashboard ${dashboardName}&body=${encodeURI(comment || '')}`;
```

Comment:
this will not work if you let users configure with `owner/repository` only :)

Reply:
yes -- i removed the other text.  the `owner/repository` flavor will fail validation